fakestorage/object.go

```diff
@@ -65,15 +65,22 @@ func (o *objectList) Swap(i int, j int) {
 //
 // If the bucket within the object doesn't exist, it also creates it. If the
 // object already exists, it overrides the object.
-func (s *Server) CreateObject(obj Object) {
-	err := s.createObject(obj)
+func (s *Server) CreateObject(obj Object) Object {
```

Comment:
Let's not change this once because it's part of the public API. We should probably deprecate this in favor of one that returns the object and an error instead of panic, but it doesn't need to be part of this PR.

Reply:
Reverted.